### PR TITLE
🔨 Add _NCS#getActivePlayers

### PR DESCRIPTION
**Pull request by LQuatre**

### CODE REVIEW

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]
Please consider moving this function into `src/api/players/client`

File: src/api/players/client/getActivePlayers.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---getPlayersServerId
+---@return void
+---@public
+function _NCS:getActivePlayers()
+    local ActivePlayers = {}
+    for k, v in pairs(GetActivePlayers()) do 
+        local found = false
+        for _, id in paurs(PlayersServerId) do 
+            if GetPlayerServerId(v) == id then
```
Comment (PABLO-1610) [inline src/api/players/client/getActivePlayers.lua:9]:
```suggestion
            if (GetPlayerServerId(v) == id) then
```

File: src/api/players/client/getActivePlayers.lua [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                found = true
+            end
+        end
+        if not found then
```
Comment (PABLO-1610) [inline src/api/players/client/getActivePlayers.lua:13]:
```suggestion
        if (not (found)) then
```

File: src/api/players/client/getActivePlayers.lua [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            table.insert(ActivePlayers, GetPlayerServerId(v))
+        end
+    end
+    return(ActivePlayers)
```
Comment (PABLO-1610) [inline src/api/players/client/getActivePlayers.lua:17]:
```suggestion
    return (ActivePlayers)
```

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]

**Comment by PABLO-1610** [comment]
I don't understand what is the purpose of the function and why it is called like this, you are looking for a specific player while calling the getActivePlayers function? Especially since we don't have any global variable named PlayersServerId.

**Comment by LQuatre** [comment]
> I don't understand what is the purpose of the function and why it is called like this, you are looking for a specific player while calling the getActivePlayers function? Especially since we don't have any global variable named PlayersServerId.

getActivePlayers was supposed to be getPlayersServerId, I renamed it, and _NCS:getActivePlayers retrieves all players present on the server (from the client) and does a sorting to remove invisible or not recognized players

**Pull request closed without merging**